packages/manager/modules/pci/src/projects/project/instances/instance/detach-private-network/detach-private-network.html

```diff
@@ -0,0 +1,35 @@
+<oui-modal
+    data-heading="{{:: 'pci_projects_project_instances_instance_detach-private-network_title' | translate}}"
+    data-primary-action="$ctrl.detachPrivateNetworks($ctrl.selectedPrivateNetworks)"
+    data-primary-label="{{:: 'pci_projects_project_instances_instance_detach-private-network_submit_label' | translate }}"
+    data-primary-disabled="$ctrl.isLoading || $ctrl.selectedPrivateNetworks.length === 0"
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{:: 'pci_projects_project_instances_instance_detach-private-network_cancel_label' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.isLoading"
+>
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <div data-ng-if="$ctrl.instance.privateNetworks.length === 0">
+        <oui-message data-type="warning">
+            <span
+                data-translate="pci_projects_project_instances_instance_detach-private-network_error_no_compatible_network"
+            ></span>
+        </oui-message>
+    </div>
+    <div data-ng-if="$ctrl.instance.privateNetworks.length > 0">
+        <oui-select
+            data-name="selectedPrivateNetworks"
+            data-model="$ctrl.selectedPrivateNetworks"
+            data-placeholder="{{ :: 'pci_projects_project_instances_instance_detach-private-network_network_placeholder' | translate }}"
+            data-items="$ctrl.instance.privateNetworks"
+            data-match="name"
+            data-searchable
+            data-multiple
+            data-disabled="$ctrl.isLoading"
```

Comment:
may be we could add a spinner during the loading time.
better customer experience i guess.

Reply:
It would be in fact a good idea.
Il check with the PO in order to create a ticket in order to add a spinner to "Attach network" and "Detach network" popups.